docs/static/releasenotes.asciidoc

```diff
@@ -31,18 +33,48 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha2,Logstash 8.0.0-alpha2>>
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
+
+[[logstash-8-6-2]]
+=== Logstash 8.6.2 Release Notes
+
+[[dependencies-8.6.2]]
+==== Updates to dependencies
+* Updated JRuby to 9.3.10.0 https://github.com/elastic/logstash/pull/14865[#14865]
+* Updated bundled JDK to 17.0.6+10 https://github.com/elastic/logstash/pull/14855[#14855]
+
+==== Plugins
+
+*Fingerprint Filter - 3.4.2*
+
+* Key config type changed to `password` type for better protection from leaks. https://github.com/logstash-plugins/logstash-filter-fingerprint/pull/71[#71]
+
+*Aws Integration - 7.0.1*
+
+* Resolved race conditions in the S3 Output plugin when handling temporary files https://github.com/logstash-plugins/logstash-integration-aws/pull/19[#19]
+
+*Elasticsearch Output - 11.12.4*
+
+* Changed `manage_template` default value to `false` when data streams is enabled https://github.com/logstash-plugins/logstash-output-elasticsearch/pull/1111[#1111]
+
+[[logstash-8-6-1]]
+=== Logstash 8.6.1 Release Notes
+
+[[dependencies-8.6.1]]
+==== Updates to dependencies
+* Updated snakeyaml to 1.33 https://github.com/elastic/logstash/pull/14848[#14848]
+
 [[logstash-8-6-0]]
 === Logstash 8.6.0 Release Notes
 
 [[features-8.6.0]]
 ==== New features and enhancements
 
 * Extends the flow rates introduced to the Node Stats API in 8.5.0 (which included windows for `current` and `lifetime`)
-to include a Technology Preview of several additional windows such as `last_15_minutes`, `last_24_hours`, etc..
-https://github.com/elastic/logstash/pull/14571[#14571]
+  to include a Technology Preview of several additional windows such as `last_15_minutes`, `last_24_hours`, etc..
+  https://github.com/elastic/logstash/pull/14571[#14571]
```

Comment:
The spaces and the following weren't present in the origin al PR #14896

Reply:
Yes and I was surprised when cherry picking that somehow this change lost, here is [the origin change](https://github.com/elastic/logstash/pull/14762/files)

Reply:
When `8.6.0` release notes was forward ported, those spaces wasn't moved also check [this](https://github.com/elastic/logstash/pull/14833/files#diff-51d73a6b1811f83bad13ef62e2d88207e0f028ef70e3e9b968ea411bc817f69cR42) for example.

Reply:
Good catch